zipline/algorithm.py

```diff
@@ -497,28 +497,33 @@ def _create_clock(self):
         trading_o_and_c = self.trading_calendar.schedule.ix[
             self.sim_params.sessions]
         market_closes = trading_o_and_c['market_close'].values.astype(np.int64)
+        minutely_emission = False
```

Comment:
Looks like this value is only used in the `else`.  Could move it there for parallel constructions between the `if` and the `else`, but not a big deal.